python/qemu/ot/util/recfmt.py

```diff
@@ -195,21 +210,19 @@ class SRecParser(RecordParser):
     def _get_next_chunk(self):
         # test if the file size can be found...
         try:
-            import sys
             self._bytes = stat(self._src.name)[6]
         except Exception:
             pass
         bc = 0
         try:
-            for (l, line) in enumerate(self._src, start=1):
+            for (lno, line) in enumerate(self._src, start=1):
                 line = line.strip()
                 if self._verbose and self._bytes:
                     opc = (50*bc)//self._bytes
                     bc += len(line)
                     pc = (50*bc)//self._bytes
```

Comment:
Maybe add spaces around operators for `opc` and `pc`? (same remark in other parsers)